News-Android-App/src/main/java/de/luhmer/owncloudnewsreader/NewsReaderListActivity.java

```diff
@@ -168,6 +168,7 @@ protected void onCreate(Bundle savedInstanceState) {
             PreferenceManager.setDefaultValues(this, R.xml.pref_notification, true);
         }
 
+        setTheme(R.style.AppTheme);
```

Comment:
Do we need to set the theme here explicitly? The PodcastActivity will set the correct theme, right? (Didn't test it yet.. )
```suggestion

```